test/model.querying.test.js

```diff
@@ -2399,6 +2399,19 @@ describe('model: querying:', function() {
       });
     });
 
+    it('removes the __v property if versionKey: false is set (gh-8934)', function(done) {
```

Comment:
We need to add a test-case for the [custom `versionKey`](https://mongoosejs.com/docs/guide.html#versionKey) as well.